spec/codecs/nmap_spec.rb

```diff
@@ -27,6 +27,15 @@
           expect(event).to be_a(LogStash::Event)
         end
       end
+
+      let(:ids) { subject.map {|e| e["id"] } }
+      it "should add a unique id field to all events" do
+        expect(ids).to eql(ids.uniq)
+      end
+
+      it "should not have any null id fields" do
+        expect(ids.include?(nil)).to be_falsey
```

Comment:
Not sure, but semantically we're checking for falseyness, which works just as well, and is more semantically correct. We don't care if its `false` proper, just something that's `falsey`.